multiqc/modules/checkatlas/checkatlas.py

```diff
@@ -0,0 +1,544 @@
+    "checkatlas/qc": "checkatlas_qc",
+    "checkatlas/umap": "_checkatlas_umap",
+    "checkatlas/tsne": "checkatlas_tsne",
+    "checkatlas/cluster": "_checkatlas_mcluster",
+    "checkatlas/annotation": "checkatlas_mannot",
+    "checkatlas/dimred": "_checkatlas_mdimred",
+    "checkatlas/specificity": "checkatlas_mspecificity",
+}
+
+
+openpng_html_script = """
+        <script>
+        function openPNG(evt, pngName, tablinks_id, tabcontent_id) {
+          var i, tabcontent, tablinks;
+          tabcontent = document.getElementsByClassName(tabcontent_id);
+          for (i = 0; i < tabcontent.length; i++) {
+            tabcontent[i].style.display = "none";
+          }
+          tablinks = document.getElementsByClassName(tablinks_id);
+          for (i = 0; i < tablinks.length; i++) {
+            tablinks[i].className = tablinks[i].className.replace(" active", "");
+          }
+          document.getElementById(pngName).style.display = "block";
+          evt.currentTarget.className += " active";
+        }
+        </script>
+        """
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """checkatlas module, parses stderr logs."""
+
+    def __init__(self):
+
+        fig_dir = os.path.join(config.data_dir, FIG_PATH)
+        os.mkdir(fig_dir)
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name="CheckAtlas",
+            anchor="checkatlas",
+            href="https://github.com/becavin-lab/checkatlas",
+            info="A one-liner tool for quality control of your single-cell atlases.",
+            doi="",
+        )
+
+        self.data_summary = dict()
+        for f in self.find_log_files("checkatlas/summary"):
+            input_fname = f["s_name"].replace("_checkatlas_summ", "")
+            s_name = self.clean_s_name(input_fname, f)
+            self.data_summary[s_name] = parse_firstline_table_logs(f["f"])
+            self.add_data_source(f, s_name)
+
+        self.data_adata = dict()
+        for f in self.find_log_files("checkatlas/adata"):
+            input_fname = f["s_name"].replace("_checkatlas_adata", "")
+            s_name = self.clean_s_name(input_fname, f)
+            self.data_adata[s_name] = parse_firstline_table_logs(f["f"])
+            self.add_data_source(f, s_name)
+
+        self.data_qc_fig = dict()
+        for f in self.find_log_files("checkatlas/qc_fig"):
+            # copy figures to multiqc data folder
+            shutil.copy2(os.path.join(f["root"], f["fn"]), fig_dir)
+            input_fname = f["s_name"].replace("_checkatlas_qc", "")
+            s_name = self.clean_s_name(input_fname, f)
+            self.data_qc_fig[s_name] = f["fn"]
+            self.add_data_source(f, s_name)
+
+        self.data_qc_counts = dict()
+        self.data_qc_genes = dict()
+        self.data_qc_mito = dict()
+        for f in self.find_log_files("checkatlas/qc"):
+            input_fname = f["s_name"].replace("_checkatlas_qc", "")
```

Comment:
Please add to `fn_clean_exts` instead ([docs](https://multiqc.info/docs/#sample-name-cleaning)).

Reply:
Removed